srcpkgs/cjs/template

```diff
@@ -1,26 +1,27 @@
 # Template file for 'cjs'
 pkgname=cjs
-version=5.4.1
+version=5.8.0
 revision=1
 build_style=meson
-build_helper=gir
-# XXX: disable tests because they need dbus, X and GTK, same as gjs
-configure_args="-Dskip_dbus_tests=true -Dskip_gtk_tests=true
- -Dprofiler=disabled -Dinstalled_tests=false"
-hostmakedepends="pkg-config gobject-introspection glib-devel dbus mozjs78-devel"
-makedepends="dbus-glib-devel readline-devel gobject-introspection
- gtk+3-devel mozjs78 mozjs78-devel"
+build_helper="gir"
+configure_args="-Dprofiler=disabled -Dinstalled_tests=false"
+hostmakedepends="pkg-config glib-devel"
+makedepends="readline-devel mozjs102-devel"
+depends="gtk+3"
+checkdepends="dbus xvfb-run gtk+3"
 short_desc="Javascript for Cinnamon"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="LGPL-2.0-or-later"
 homepage="https://github.com/linuxmint/cjs"
 distfiles="https://github.com/linuxmint/cjs/archive/refs/tags/${version}.tar.gz"
-checksum=212fa302f15ea955af6dc87fdba3898f751d078df91cb84b0e6615d5a2b84e15
-# ΧΧ A single test fails on x86_64,  meson reports in the configure stage:
-# meson.build:663: WARNING: Your libc does not have the C.UTF-8 locale and no
-# other suitable UTF-8 fallback locale could be found. You can still build GJS,
-# but some tests will fail.
-make_check=extended
+checksum=04e7c10a0a4d88ba446e6fb4f4788f732984e0318d775c646634a8619dfff7d2
+make_check_pre="xvfb-run"
+
+if [ "$XBPS_CHECK_PKGS" ]; then
+	configure_args+=" -Dskip_dbus_tests=false -Dskip_gtk_tests=false"
+else
+	configure_args+=" -Dskip_dbus_tests=true -Dskip_gtk_tests=true"
+fi
```

Comment:
Do not skip anything. Just build them all.

Reply:
Without them, buliding without tests fails with a `meson.build:272:19: ERROR: Program 'dbus-run-session' not found or not executable` error as can be seen in the CI, so I think they ought to be kept (at least the `-Dskip_dbus_tests=true`, I haven't figured out if the gtk parameter is needed too). The gjs template also does a similar thing.

Reply:
I meant add those `checkdepends` into `hostmakedepends`